apps/app/src/views/routes/NewOrder/components/SelectPharmacyCard/index.tsx

```diff
@@ -135,12 +133,6 @@ export const SelectPharmacyCard: React.FC<SelectPharmacyCardProps> = ({
         />
       )
     }
-    // {
```

Comment:
this is still supposed to be commented out?

Reply:
nope, removed since send to patient is second in the list